cmd/crictl/image.go

```diff
@@ -164,6 +171,27 @@ var listImageCommand = &cli.Command{
 			}
 		}
 		for _, image := range r.Images {
+			// don't show intermediate images if showall is not set
+			if !showAll {
+				status, err := ImageStatus(imageClient, image.Id, true)
+				if err != nil {
+					return err
+				}
+
+				if status.GetInfo() != nil {
+					var info map[string]interface{}
+					err := json.Unmarshal([]byte(status.Info["info"]), &info)
+					if err != nil {
+						return err
```

Comment:
this and other places - it may be useful to give additional details by rapping this error. Perhaps the image name and maybe the `status.Info` string?